wgengine/magicsock/derp.go

```diff
@@ -136,10 +135,7 @@ func (c *Conn) pickDERPFallback() int {
 		return pickDERPFallbackForTests()
 	}
 
-	// TODO: use math/rand/v2 here.
-	h := fnv.New64()
-	fmt.Fprintf(h, "%p/%d", c, processStartUnixNano) // arbitrary
-	return ids[rand.New(rand.NewSource(int64(h.Sum64()))).Intn(len(ids))]
+	return ids[rands.IntN(uint64(processStartUnixNano), len(ids))]
```

Comment:
this will always return exactly the same value for the entire lifetime of the process

Reply:
the code was previously intending to (i think) return a different value if/when, and only when the conn object changes.

i think processStartUnixNano is entirely superfluous here and can be dropped.